posthog/api/alert.py

```diff
@@ -73,6 +75,11 @@ def validate(self, data):
         return data
 
 
+class RelativeDateTimeField(serializers.DateTimeField):
```

Comment:
I'm not a Django expert so I'm not sure i really get this - what is this doing?

Reply:
Ahh yes, this is a bit annoying, `snoozed_until` is a date time field, so when we call the update endpoint, we are forced to send a datetime string. But we don't want this as our convention is to send `+4h`, `-7d` (the time offset/relative date range) so that the backend can then use the user's timezone to compute the time to snooze until. To support being able to send these relative date time strings as value for `snoozed_until` to the API, had to write a custom serializer

Reply:
actually now that I say this, need to make sure to always store `snoozed_until` as UTC as we compare `snoozed_until` < utc.now() 😅